rules/integrations/aws/credential_access_new_terms_secretsmanager_getsecretvalue.toml

```diff
@@ -93,10 +93,12 @@ tags = [
     "Investigation Guide",
 ]
 timestamp_override = "event.ingested"
-type = "query"
+type = "new_terms"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:secretsmanager.amazonaws.com and event.action:GetSecretValue
+event.dataset:aws.cloudtrail and event.provider:secretsmanager.amazonaws.com and
+    event.action:GetSecretValue and event.outcome:success and
+    not user_agent.name: ("Chrome" or "Firefox" or "Safari" or "Edge" or "Brave" or "Opera" or "aws-cli")
```

Comment:
user agent can be easily spoofed ? if that's the case maybe drop this exclusion, NewTerm should take care of it if it's a recurrent FP.

Reply:
Thanks Samir! Yea from a tooling perspective, the user-agent could be spoofed easily. The reason I exclude browsers and aws-cli is because of the libraries and SDKs AWS makes available. If used, for example in the python application I made below, the user-agent is `boto`. While these are legit SDKs, they still suggest either a malicious tool or a one-off script/tool where the UA was not changed. I'd expect in a more advanced intrusion, the actor would cover their footprint more, but in some cases may not :)

<img width="836" alt="Screenshot 2023-05-03 at 9 19 13 AM" src="https://user-images.githubusercontent.com/99630311/235928123-5b8d6f6e-e3ed-43f8-beb7-74efc2ee3fc3.png">